gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/DiagnosticReportMapperTest.java

```diff
@@ -294,6 +295,156 @@ public void When_DiagnosticReportContainsChildObservationComment_Expect_Observat
+                        <NarrativeStatement classCode="OBS" moodCode="EVN">
+                            <id root="21B95C86-121F-4BC5-A2CD-3E2955F4D604" />
+                            <text>Option C</text>
+                            <statusCode code="COMPLETE" />
+                            <availabilityTime value="201101111533" />
+                        </NarrativeStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <ObservationStatement classCode="OBS" moodCode="EVN">
+                            <id root="E0DD7160-64CA-48F3-AB00-B21C39D3FB23" />
+                            <code code="136Q.00" codeSystem="2.16.840.1.113883.2.1.6.2" displayName="Very heavy drinker">
+                                <translation code="228279004" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" />
+                            </code>
+                            <statusCode code="COMPLETE" />
+                            <effectiveTime>
+                                <center nullFlavor="NI" />
+                            </effectiveTime>
+                            <availabilityTime value="201101111533" />
+                        </ObservationStatement>
+                    </component>
+                    <component typeCode="COMP" contextConductionInd="true">
+                        <NarrativeStatement classCode="OBS" moodCode="EVN">
+                            <id root="B141FC3E-4DD2-4412-9655-DF43BBCE290B" />
+                            <text>Some freetext - This is total freetext madness</text>
+                            <statusCode code="COMPLETE" />
+                            <availabilityTime value="201101111533" />
+                        </NarrativeStatement>
+                    </component>
+               </CompoundStatement>
+               """);
+
+        var ehrExtract = unmarshallEhrExtractFromXmlString(inputXml);
+
+        var diagnosticReports = diagnosticReportMapper.mapResources(
+            ehrExtract,
+            PATIENT,
+            createEncounterList(),
+            PRACTICE_CODE);
+        var diagnosticReport = diagnosticReports.get(0);
+
+        assertAll(
+            () -> assertThat(diagnosticReport.getResult()).hasSize(FOUR),
+            () -> assertEquals("Observation/3E1A4EA2-661B-4467-8843-6A6B21DEF14F",
+                               diagnosticReport.getResult().get(0).getReference()),
+            () -> assertEquals("Observation/E8746411-9D21-4A5C-B70B-9D1FF00D3AE4",
+                               diagnosticReport.getResult().get(1).getReference()),
+            () -> assertEquals("Observation/21B95C86-121F-4BC5-A2CD-3E2955F4D604",
+                               diagnosticReport.getResult().get(2).getReference()),
+            () -> assertEquals("Observation/B141FC3E-4DD2-4412-9655-DF43BBCE290B",
+                               diagnosticReport.getResult().get(THREE).getReference())
+        );
```

Comment:
Couldn't this be simplified down to something like ` assertThat(diagnosticReport.getResult.map(r -> r.getReference()).isEqualTo(List.of("Observation/3E1A4EA2-661B-4467-8843-6A6B21DEF14F", "Observation/E8746411-9D21-4A5C-B70B-9D1FF00D3AE4", "Observation/21B95C86-121F-4BC5-A2CD-3E2955F4D604", "Observation/B141FC3E-4DD2-4412-9655-DF43BBCE290B"))` ?

Reply:
yes, it can be, thanks